src/library/simplified/extension/checkout-date.js

```diff
@@ -5,6 +5,10 @@ if (utag.isHSR() && b['entity.hotels.search.hotelParameters.checkOutDate'])
 {
     b['checkOutDate'] = b['entity.hotels.search.hotelParameters.checkOutDate'];
 }
+else if (utag.isRailSearchResults() && b['entity.railSearch.searchParameters.journeyLegs.1']!= undefined )
+{
+    b["checkOutDate"] = b['entity.railSearch.searchParameters.journeyLegs.1'].isoFormatDate;
```

Comment:
isoFormatDate  existis in UDO. Please have a look at the screenshot.

![image](https://cloud.githubusercontent.com/assets/21168911/24895686/5634786e-1eaf-11e7-923d-b1599a74d130.png)